src/globalStyles/global.scss

```diff
@@ -129,3 +129,9 @@ th.reactable-header-sort-asc:after {
 .table {
   margin-bottom: 5px;
 }
+
+.hiddenByPosition {
+  position: absolute;
```

Comment:
heh.   i think the fact that it happens only when tab renders in display:none indicates it a really nasty issue.   we could bust our asses to see if we can fix it (it might not be fixable) but then it will surely happen again in some other visualization down the road.  it's pretty common to hide things in this fashion.  i think this is actually relatively elegant.